### PR TITLE
Add Thin Egress App support to example

### PR DESCRIPTION
**Pull request by yjpa7145**
This is part of [CUMULUS-1321 - As a Cumulus User, I want to deploy Cumulus with ASF's thin egress app as the distribution API](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1321).  This PR adds the ability to deploy the Thin Egress App as part of our integration stacks.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ We have encountered transient lambda service errors in our integration testing.
   - Added additional `ems` configuration parameters for sending the ingest reports to EMS
   - Added functionality to send daily ingest reports to EMS
 
+### Fixed
+
+- **CUMULUS-1319**
+  - Fixed a bug where granule ingest times were not being stored to the database
+
```
Comment (Jkovarik) [inline CHANGELOG.md]: This seems like it's not the right changelog entry?

File: example/README.md [hunk 2]
```diff
@@ -27,6 +27,200 @@ DEPLOYMENT=cumulus-from-source jasmine spec/ingestGranule/IngestGranuleSuccessSp
...
+
+Store the secrets to Secrets Manager, substituting the appropriate values for
+`UrsId` and `UrsAuth`.
+
+```shell
+aws secretsmanager create-secret \
+    --name my-tea-urs-creds \
+    --secret-string '{"UrsId": "client-id","UrsAuth": "base64-encoded-value"}' \
+    --description 'URS Credentials for My TEA Distribution App'
+```
```
Comment (flamingbear) [inline example/README.md]: Something isn't crystal clear here.  It took me a couple of reads to figure out that the base64-encoded-value should be the output of L116.
Maybe clarifying L112 with the `The client id and password need to be joined and 
base64 encoded. before passed as the UrsAuth value.` . Or something like that.  And If I still have the wrong idea, of what has to happen, it also says it's not clear.

Comment (yjpa7145) [reply example/README.md]: Let me know if this is better: https://github.com/nasa/cumulus/pull/1061/files#diff-20bfc3ff6a860483887b93bf9cf0d135R110

Comment (flamingbear) [reply example/README.md]: 1d44138 is 💯   Thanks.

**Pull request merged**